doc/how_to_validate.rst

```diff
@@ -0,0 +1,64 @@
+..
+  SPDX-FileCopyrightText: 2021 The PyPSA meets Earth authors
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _tutorial::
+
+##########################################
+How to validate?
+##########################################
+
+.. TODO add a list of actions needed to do the validation
+
+Data workflow is PyPSA-Earth allows to extract all the data needed to build your energy model from global open data sources. An example of the validation procedure is available in the `Nigeria validation <https://github.com/pypsa-meets-earth/documentation/blob/main/notebooks/validation/validation_nigeria.ipynb>`_ notebook. Public information on the power system of Nigeria are compared to those obtained from the PyPSA-Earth model.
```

Comment:
I'd remove the first sentence and be direct on what this section is about.
Like "To validate the data obtained with PyPSA-Earth, we recommend to go through the procedure here detailed. An exampled of ..."

Reply:
Revised